maintenance_internal.conf

```diff
@@ -0,0 +1,8 @@
+error_page 503 @maintenance;
+recursive_error_pages on;
+location @maintenance {
+    error_page 405 = /index.html;
+    add_header Cache-Control no-cache;
+    root /usr/share/nginx/html/maintenance;
```

Comment:
Maybe one page is enough and root is redundant.
Had thoughts if somebody needs a place for css

Reply:
Thanks! I really need it, and wait for merge it!